addons/website/views/snippets/s_quotes_carousel.xml

```diff
@@ -2,8 +2,8 @@
 <odoo>
 
 <template id="s_quotes_carousel" name="Quotes">
-    <section class="s_quotes_carousel_wrapper" data-vxml="001">
-        <div id="myQuoteCarousel" class="s_quotes_carousel s_carousel_default carousel slide" data-interval="10000">
+    <section class="s_quotes_carousel_wrapper" data-vxml="001" data-vcss="001">
+        <div id="myQuoteCarousel" class="s_quotes_carousel s_carousel_default carousel slide bg-200" data-interval="10000">
```

Comment:
To check with the new color system, I removed the possibility to add color class on the standard carousel... probably should be the same here (just inline color).